.jenkins/continuous.groovy

```diff
@@ -176,7 +176,7 @@ pipeline {
                                     -D CMAKE_CXX_COMPILER_LAUNCHER=ccache \
                                     -D CMAKE_CXX_COMPILER=clang++ \
                                     -D CMAKE_CXX_EXTENSIONS=OFF \
-                                    -D CMAKE_CXX_FLAGS="-Wpedantic -Wall -Wextra" \
+                                    -D CMAKE_CXX_FLAGS="-Wpedantic -Wall -Wextra -DCUB_USE_COOPERATIVE_GROUPS" \
```

Comment:
Why this change?

Reply:
The full commit message for 034ec05b32deb6d089a5d289359dc1b47da385a3 is
>Workaround problematic __syncs in Thrust.
>
> See https://bugs.llvm.org/show_bug.cgi?id=43505

Without this, `Thrust` is using `shlf` without sync which was removed from PTX ISA version 6.4. According to the same bug report, this should be fixed in `Cuda 11.3` but I preferred the workaround over figuring out how to get a recent enough clang for that.

Reply:
Technically, this preprocessor variable is not documented and I'm not quite sure we should always mess with it but sure, I'll give it a try and add it.

Reply:
https://github.com/NVIDIA/cub/pull/164

Reply:
> We need to do that workaround in our headers

Agreed.